src/components/flex-container/flex-container.scss

```diff
@@ -49,3 +46,14 @@
 :host(limel-flex-container[justify='space-evenly']) {
     justify-content: space-evenly;
 }
+
+// Since consumers often copy/paste from examples of `limel-dialog` from the
+// documentations, they often put their buttons in a `limel-flex-container`
+// which is used in the footer slot of those examples with buttons.
+// So to prevent unwanted layout problems of buttons having no distance, we use
+// below code to target `limel-flex-container`s in those slots.
+:host(limel-flex-container[slot='button']) {
+    gap: 0.5rem; // Makes sure buttons get some default distance
+    width: 100%;
+    justify-content: flex-end;
+}
```

Comment:
@jgroth I dug through the history to find where limel-flex-container was added, and it was added way back in [v17.0.0](https://github.com/Lundalogik/lime-elements/releases/tag/v17.0.0), without reference to any issue, or anything else really explaining what the purpose was. It was added in a commit by you, so I'm wondering if you remember any details? 🙂

Reply:
Haha no sorry I don't remember :smile: I think we just wanted some element for the consultants that they could use if they did not know a lot of CSS

Reply:
Yeah… and they probably like it! But I don't… 😆

Or, well, I'm just worried it will be causing us trouble… I feel like we should at least try not to build ourselves further into it by making it more or less required in dialog footers… 🤔